plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -61,9 +61,6 @@
         id="com.google.cloud.tools.eclipse"
         name="Google Cloud Platform"
         class="com.google.cloud.tools.eclipse.properties.GooglePropertyPage">
-       <enabledWhen>
-         <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
```

Comment:
I'll restrict it to projects. Non-faceted projects still have the `Project Facets` page in their preferences, so if the user opens that, converts it to faceted, then adds the GAE Standard (or Flexible) facet, they can immediately configure deployment settings if they want to.